packages/flutter/lib/src/material/bottom_navigation_bar.dart

```diff
@@ -57,91 +57,91 @@ class DestinationLabel {
     assert(this.title != null);
   }
 
-  /// The icon of the label.
+  /// The icon of the item.
   ///
   /// Typically the icon is an [Icon] or an [IconImage] widget. If another type
   /// of widget is provided then it should configure itself to match the current
   /// [IconTheme] size and color.
   final Widget icon;
 
-  /// The title of the label.
+  /// The title of the item.
   final Widget title;
 
   /// The color of the background radial animation.
   ///
   /// If the navigation bar's type is [BottomNavigationBarType.shifting], then
-  /// the entire bar is flooded with the [backgroundColor] when this label is
+  /// the entire bar is flooded with the [backgroundColor] when this item is
   /// tapped.
   final Color backgroundColor;
 }
 
 /// A material widget displayed at the bottom of an app for selecting among a
 /// small number of views.
 ///
-/// The bottom navigation bar consists of multiple destinations in the form of
-/// labels laid out on top of a piece of material. It provies quick navigation
-/// between top-level views of an app and is typically used on mobile. For
-/// larger screens, side navigation may be a better fit.
+/// The bottom navigation bar consists of multiple items in the form of
+/// labels, icons, or both laid out on top of a piece of material. It provides
```

Comment:
grammar nit: i'd put a comma after "both" to make it clear that the next part doesn't only apply to the "both" option.